ble/UUID.h

```diff
@@ -42,7 +97,7 @@ class UUID {
      *         different service or characteristics on the BLE device.
      *
      * @param longUUID
-     *          The 128-bit (16-byte) UUID value, MSB first (big-endian).
+     *          The 128-bit (16-byte) UUID value, LSB first (little-endian).
```

Comment:
I see your point. My main goal with this PR is to have the internal representation in LSB and not having to change the API would be preferred.

I mainly changed the constructor so that getBaseUUID will return the same array as it was constructed with.

What I would really like is a no-copy constructor that takes a ```const uint8_t*``` and returns it unaltered when getBaseUUID is called.